tools/docker/tests/flows/test_ap_config_renew.py

```diff
@@ -0,0 +1,56 @@
+from . import BaseTest, LogType, DeviceType
+import time
+"""
+Original shell test:
```

Comment:
I don't think it's very useful to repeat that here. Well, actually for the review it is :-) but when it's going to be merged, it looses its purpose.

So I propose to keep it here, but add another commit that removes it again.

In a similar vein, I think it would be useful to split this PR into a first commit that adds the infrastructure (`flows.__init__.py` and `run_tests.py`), and then two commits that add the two tests. The people who will need to write tests just need to review those latter commits.